sdk/src/beta9/__init__.py

```diff
@@ -23,4 +24,5 @@
     "env",
     "GpuType",
     "PythonVersion",
+    "Output",
```

Comment:
Reminder to update beam-client after we release this. Also we should bump the sdk version in this PR.

Reply:
Bumped the SDK version just now. Will build and publish after this is merged.